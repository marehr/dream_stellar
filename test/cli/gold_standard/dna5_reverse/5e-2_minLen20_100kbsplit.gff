seq1::100000:200000	Stellar	eps-matches	82569	82589	95.238	-	.	seq2::0:100000;seq2Range=29771,29791;eValue=1.6929;cigar=21M;mutations=13A
seq1::400000:500000	Stellar	eps-matches	79518	79538	95.238	-	.	seq2::0:100000;seq2Range=83080,83100;eValue=1.6929;cigar=21M;mutations=19C
seq1::900000:1000000	Stellar	eps-matches	71517	71537	95.4545	-	.	seq2::0:100000;seq2Range=55261,55282;eValue=0.515015;cigar=9M1I12M;mutations=10T
seq1::0:100000	Stellar	eps-matches	19458	19477	95	-	.	seq2::0:100000;seq2Range=30538,30556;eValue=5.56469;cigar=7M1D12M;mutations=
seq1::200000:300000	Stellar	eps-matches	38268	38287	95.238	-	.	seq2::0:100000;seq2Range=83582,83602;eValue=1.6929;cigar=16M1I4M;mutations=17A
seq1::400000:500000	Stellar	eps-matches	86491	86510	95	-	.	seq2::0:100000;seq2Range=60990,61009;eValue=5.56469;cigar=20M;mutations=14G
seq1::400000:500000	Stellar	eps-matches	71158	71177	95.238	-	.	seq2::0:100000;seq2Range=32620,32640;eValue=1.6929;cigar=13M1I7M;mutations=14G
seq1::800000:900000	Stellar	eps-matches	5362	5381	95	-	.	seq2::0:100000;seq2Range=46863,46882;eValue=5.56469;cigar=20M;mutations=4G
seq1::200000:300000	Stellar	eps-matches	73607	73625	95	-	.	seq2::0:100000;seq2Range=9351,9370;eValue=5.56469;cigar=6M1I13M;mutations=7G
seq1::800000:900000	Stellar	eps-matches	5470	5488	95	-	.	seq2::0:100000;seq2Range=41851,41870;eValue=5.56469;cigar=12M1I7M;mutations=13G
seq1::700000:800000	Stellar	eps-matches	95116	95137	95.4545	-	.	seq2::100000:200000;seq2Range=168,188;eValue=0.515015;cigar=15M1D6M;mutations=
seq1::400000:500000	Stellar	eps-matches	94432	94451	95	-	.	seq2::100000:200000;seq2Range=9070,9088;eValue=5.56469;cigar=8M1D11M;mutations=
seq1::400000:500000	Stellar	eps-matches	78520	78539	95	-	.	seq2::100000:200000;seq2Range=372,390;eValue=5.56469;cigar=12M1D7M;mutations=
seq1::800000:900000	Stellar	eps-matches	75106	75125	95	-	.	seq2::100000:200000;seq2Range=27516,27535;eValue=5.56469;cigar=20M;mutations=9G
seq1::800000:900000	Stellar	eps-matches	60711	60730	95	-	.	seq2::100000:200000;seq2Range=37910,37928;eValue=5.56469;cigar=6M1D13M;mutations=
seq1::400000:500000	Stellar	eps-matches	18601	18619	95	-	.	seq2::100000:200000;seq2Range=59496,59515;eValue=5.56469;cigar=17M1I2M;mutations=18G
seq1::600000:700000	Stellar	eps-matches	3126	3144	95	-	.	seq2::100000:200000;seq2Range=92740,92759;eValue=5.56469;cigar=15M1I4M;mutations=16G
seq1::200000:300000	Stellar	eps-matches	59870	59892	95.6521	-	.	seq2::200000:300000;seq2Range=42055,42077;eValue=0.156679;cigar=23M;mutations=14T
seq1::500000:600000	Stellar	eps-matches	90053	90074	95.4545	-	.	seq2::200000:300000;seq2Range=45228,45249;eValue=0.515015;cigar=22M;mutations=17A
seq1::900000:1000000	Stellar	eps-matches	86804	86825	95.4545	-	.	seq2::200000:300000;seq2Range=55704,55725;eValue=0.515015;cigar=22M;mutations=9G
seq1::200000:300000	Stellar	eps-matches	15664	15684	95.238	-	.	seq2::200000:300000;seq2Range=12347,12367;eValue=1.6929;cigar=21M;mutations=10G
seq1::400000:500000	Stellar	eps-matches	4450	4469	95	-	.	seq2::200000:300000;seq2Range=36052,36070;eValue=5.56469;cigar=4M1D15M;mutations=
seq1::500000:600000	Stellar	eps-matches	55342	55361	95	-	.	seq2::200000:300000;seq2Range=44036,44054;eValue=5.56469;cigar=4M1D15M;mutations=
seq1::500000:600000	Stellar	eps-matches	29470	29489	95.238	-	.	seq2::200000:300000;seq2Range=61759,61779;eValue=1.6929;cigar=15M1I5M;mutations=16G
seq1::600000:700000	Stellar	eps-matches	45038	45057	95	-	.	seq2::200000:300000;seq2Range=34191,34209;eValue=5.56469;cigar=8M1D11M;mutations=
seq1::700000:800000	Stellar	eps-matches	63601	63620	95	-	.	seq2::200000:300000;seq2Range=45161,45179;eValue=5.56469;cigar=10M1D9M;mutations=
seq1::900000:1000000	Stellar	eps-matches	95415	95434	95	-	.	seq2::200000:300000;seq2Range=28306,28325;eValue=5.56469;cigar=20M;mutations=8G
seq1::0:100000	Stellar	eps-matches	71324	71342	95	-	.	seq2::200000:300000;seq2Range=18249,18268;eValue=5.56469;cigar=6M1I13M;mutations=7C
seq1::300000:400000	Stellar	eps-matches	46392	46410	95	-	.	seq2::200000:300000;seq2Range=46662,46681;eValue=5.56469;cigar=4M1I15M;mutations=5G
seq1::600000:700000	Stellar	eps-matches	43822	43840	95	-	.	seq2::200000:300000;seq2Range=44584,44603;eValue=5.56469;cigar=16M1I3M;mutations=17T
seq1::500000:600000	Stellar	eps-matches	63210	63231	95.4545	-	.	seq2::300000:400000;seq2Range=55253,55274;eValue=0.515015;cigar=22M;mutations=4A
seq1::400000:500000	Stellar	eps-matches	72552	72571	95	-	.	seq2::300000:400000;seq2Range=3200,3218;eValue=5.56469;cigar=7M1D12M;mutations=
seq1::800000:900000	Stellar	eps-matches	86471	86490	95	-	.	seq2::300000:400000;seq2Range=37574,37592;eValue=5.56469;cigar=16M1D3M;mutations=
seq1::700000:800000	Stellar	eps-matches	95190	95208	95	-	.	seq2::300000:400000;seq2Range=61614,61633;eValue=5.56469;cigar=8M1I11M;mutations=9T
seq1::700000:800000	Stellar	eps-matches	65718	65740	95.6521	-	.	seq2::400000:500000;seq2Range=80465,80487;eValue=0.156679;cigar=23M;mutations=15T
seq1::500000:600000	Stellar	eps-matches	76858	76878	95.238	-	.	seq2::400000:500000;seq2Range=91562,91582;eValue=1.6929;cigar=21M;mutations=3C
seq1::200000:300000	Stellar	eps-matches	77755	77774	95	-	.	seq2::400000:500000;seq2Range=77270,77288;eValue=5.56469;cigar=13M1D6M;mutations=
seq1::500000:600000	Stellar	eps-matches	84612	84631	95.238	-	.	seq2::400000:500000;seq2Range=27382,27402;eValue=1.6929;cigar=19M1I1M;mutations=20G
seq1::600000:700000	Stellar	eps-matches	77977	77996	95	-	.	seq2::400000:500000;seq2Range=39433,39452;eValue=5.56469;cigar=20M;mutations=11C
seq1::200000:300000	Stellar	eps-matches	93473	93491	95	-	.	seq2::400000:500000;seq2Range=21802,21821;eValue=5.56469;cigar=18M1I1M;mutations=19C
seq1::500000:600000	Stellar	eps-matches	8105	8123	95	-	.	seq2::400000:500000;seq2Range=207,226;eValue=5.56469;cigar=8M1I11M;mutations=9T
seq1::700000:800000	Stellar	eps-matches	63547	63565	95	-	.	seq2::400000:500000;seq2Range=69456,69475;eValue=5.56469;cigar=4M1I15M;mutations=5C
seq1::700000:800000	Stellar	eps-matches	32502	32520	95	-	.	seq2::400000:500000;seq2Range=46091,46110;eValue=5.56469;cigar=2M1I17M;mutations=3C
seq1::800000:900000	Stellar	eps-matches	87924	87944	95.238	-	.	seq2::500000:600000;seq2Range=27360,27380;eValue=1.6929;cigar=21M;mutations=7T
seq1::800000:900000	Stellar	eps-matches	14505	14525	95.238	-	.	seq2::500000:600000;seq2Range=46149,46168;eValue=1.6929;cigar=8M1D12M;mutations=
seq1::200000:300000	Stellar	eps-matches	52314	52333	95	-	.	seq2::500000:600000;seq2Range=18921,18939;eValue=5.56469;cigar=9M1D10M;mutations=
seq1::400000:500000	Stellar	eps-matches	8714	8733	95	-	.	seq2::500000:600000;seq2Range=74104,74123;eValue=5.56469;cigar=20M;mutations=6A
seq1::500000:600000	Stellar	eps-matches	89984	90003	95	-	.	seq2::500000:600000;seq2Range=62730,62749;eValue=5.56469;cigar=20M;mutations=3T
seq1::500000:600000	Stellar	eps-matches	59003	59022	95.238	-	.	seq2::500000:600000;seq2Range=45195,45215;eValue=1.6929;cigar=15M1I5M;mutations=16A
seq1::500000:600000	Stellar	eps-matches	54010	54029	95	-	.	seq2::500000:600000;seq2Range=180,199;eValue=5.56469;cigar=20M;mutations=19A
seq1::200000:300000	Stellar	eps-matches	85955	85973	95	-	.	seq2::500000:600000;seq2Range=17350,17369;eValue=5.56469;cigar=5M1I14M;mutations=6G
seq1::900000:1000000	Stellar	eps-matches	54644	54662	95	-	.	seq2::500000:600000;seq2Range=66182,66201;eValue=5.56469;cigar=3M1I16M;mutations=4A
seq1::900000:1000000	Stellar	eps-matches	5577	5595	95	-	.	seq2::500000:600000;seq2Range=28544,28563;eValue=5.56469;cigar=16M1I3M;mutations=17G
seq1::500000:600000	Stellar	eps-matches	58950	58970	95.238	-	.	seq2::600000:700000;seq2Range=83633,83652;eValue=1.6929;cigar=12M1D8M;mutations=
seq1::900000:1000000	Stellar	eps-matches	60203	60223	95.238	-	.	seq2::600000:700000;seq2Range=17579,17598;eValue=1.6929;cigar=19M1D1M;mutations=
seq1::300000:400000	Stellar	eps-matches	62963	62982	95	-	.	seq2::600000:700000;seq2Range=18522,18540;eValue=5.56469;cigar=12M1D7M;mutations=
seq1::400000:500000	Stellar	eps-matches	55364	55383	95.238	-	.	seq2::600000:700000;seq2Range=69817,69837;eValue=1.6929;cigar=5M1I15M;mutations=6C
seq1::400000:500000	Stellar	eps-matches	42608	42627	95	-	.	seq2::600000:700000;seq2Range=41357,41375;eValue=5.56469;cigar=7M1D12M;mutations=
seq1::500000:600000	Stellar	eps-matches	63144	63163	95	-	.	seq2::600000:700000;seq2Range=3709,3727;eValue=5.56469;cigar=13M1D6M;mutations=
seq1::600000:700000	Stellar	eps-matches	44858	44877	95	-	.	seq2::600000:700000;seq2Range=76277,76295;eValue=5.56469;cigar=7M1D12M;mutations=
seq1::600000:700000	Stellar	eps-matches	41510	41529	95	-	.	seq2::600000:700000;seq2Range=7675,7693;eValue=5.56469;cigar=13M1D6M;mutations=
seq1::600000:700000	Stellar	eps-matches	20345	20364	95	-	.	seq2::600000:700000;seq2Range=70708,70727;eValue=5.56469;cigar=20M;mutations=16G
seq1::0:100000	Stellar	eps-matches	3331	3349	95	-	.	seq2::600000:700000;seq2Range=31237,31256;eValue=5.56469;cigar=9M1I10M;mutations=10T
seq1::200000:300000	Stellar	eps-matches	16605	16623	95	-	.	seq2::600000:700000;seq2Range=69253,69272;eValue=5.56469;cigar=9M1I10M;mutations=10T
seq1::700000:800000	Stellar	eps-matches	21801	21821	95.4545	-	.	seq2::700000:800000;seq2Range=39398,39419;eValue=0.515015;cigar=10M1I11M;mutations=11C
seq1::0:100000	Stellar	eps-matches	77396	77415	95	-	.	seq2::700000:800000;seq2Range=85298,85316;eValue=5.56469;cigar=7M1D12M;mutations=
seq1::0:100000	Stellar	eps-matches	60971	60990	95	-	.	seq2::700000:800000;seq2Range=60087,60106;eValue=5.56469;cigar=20M;mutations=13T
seq1::100000:200000	Stellar	eps-matches	23831	23850	95	-	.	seq2::700000:800000;seq2Range=91401,91420;eValue=5.56469;cigar=20M;mutations=15A
seq1::400000:500000	Stellar	eps-matches	28970	28989	95	-	.	seq2::700000:800000;seq2Range=91789,91807;eValue=5.56469;cigar=6M1D13M;mutations=
seq1::700000:800000	Stellar	eps-matches	82079	82098	95	-	.	seq2::700000:800000;seq2Range=44396,44414;eValue=5.56469;cigar=8M1D11M;mutations=
seq1::700000:800000	Stellar	eps-matches	15611	15630	95	-	.	seq2::700000:800000;seq2Range=69274,69293;eValue=5.56469;cigar=20M;mutations=11A
seq1::900000:1000000	Stellar	eps-matches	15539	15558	95.238	-	.	seq2::700000:800000;seq2Range=99352,99372;eValue=1.6929;cigar=8M1I12M;mutations=9C
seq1::400000:500000	Stellar	eps-matches	33635	33655	95.238	-	.	seq2::800000:900000;seq2Range=79014,79033;eValue=1.6929;cigar=2M1D18M;mutations=
seq1::0:100000	Stellar	eps-matches	90438	90457	95	-	.	seq2::800000:900000;seq2Range=77934,77953;eValue=5.56469;cigar=20M;mutations=6G
seq1::0:100000	Stellar	eps-matches	66376	66395	95	-	.	seq2::800000:900000;seq2Range=42457,42475;eValue=5.56469;cigar=9M1D10M;mutations=
seq1::0:100000	Stellar	eps-matches	26503	26522	95	-	.	seq2::800000:900000;seq2Range=42546,42565;eValue=5.56469;cigar=20M;mutations=9C
seq1::400000:500000	Stellar	eps-matches	79793	79812	95	-	.	seq2::800000:900000;seq2Range=53178,53197;eValue=5.56469;cigar=20M;mutations=12C
seq1::500000:600000	Stellar	eps-matches	79568	79587	95	-	.	seq2::800000:900000;seq2Range=94330,94348;eValue=5.56469;cigar=19M1D;mutations=
seq1::600000:700000	Stellar	eps-matches	98067	98086	95	-	.	seq2::800000:900000;seq2Range=72432,72451;eValue=5.56469;cigar=20M;mutations=4G
seq1::600000:700000	Stellar	eps-matches	87547	87566	95	-	.	seq2::800000:900000;seq2Range=70365,70383;eValue=5.56469;cigar=10M1D9M;mutations=
seq1::600000:700000	Stellar	eps-matches	73645	73664	95	-	.	seq2::800000:900000;seq2Range=74398,74416;eValue=5.56469;cigar=14M1D5M;mutations=
seq1::600000:700000	Stellar	eps-matches	44813	44832	95	-	.	seq2::800000:900000;seq2Range=65801,65819;eValue=5.56469;cigar=17M1D2M;mutations=
seq1::700000:800000	Stellar	eps-matches	61138	61157	95	-	.	seq2::800000:900000;seq2Range=49566,49585;eValue=5.56469;cigar=20M;mutations=7C
seq1::900000:1000000	Stellar	eps-matches	30403	30422	95	-	.	seq2::800000:900000;seq2Range=32382,32400;eValue=5.56469;cigar=15M1D4M;mutations=
seq1::400000:500000	Stellar	eps-matches	47396	47414	95	-	.	seq2::800000:900000;seq2Range=28387,28406;eValue=5.56469;cigar=17M1I2M;mutations=18A
seq1::400000:500000	Stellar	eps-matches	24852	24870	95	-	.	seq2::800000:900000;seq2Range=82879,82898;eValue=5.56469;cigar=13M1I6M;mutations=14T
seq1::300000:400000	Stellar	eps-matches	53237	53258	95.4545	-	.	seq2::900000:1000000;seq2Range=5108,5129;eValue=0.515015;cigar=22M;mutations=3G
seq1::0:100000	Stellar	eps-matches	53898	53917	95	-	.	seq2::900000:1000000;seq2Range=49225,49244;eValue=5.56469;cigar=20M;mutations=10G
seq1::0:100000	Stellar	eps-matches	31264	31283	95	-	.	seq2::900000:1000000;seq2Range=93852,93871;eValue=5.56469;cigar=20M;mutations=4C
seq1::300000:400000	Stellar	eps-matches	77549	77568	95	-	.	seq2::900000:1000000;seq2Range=20264,20282;eValue=5.56469;cigar=16M1D3M;mutations=
seq1::300000:400000	Stellar	eps-matches	16447	16466	95.238	-	.	seq2::900000:1000000;seq2Range=62072,62092;eValue=1.6929;cigar=9M1I11M;mutations=10A
seq1::400000:500000	Stellar	eps-matches	2092	2111	95	-	.	seq2::900000:1000000;seq2Range=28807,28825;eValue=5.56469;cigar=6M1D13M;mutations=
seq1::800000:900000	Stellar	eps-matches	57248	57267	95.238	-	.	seq2::900000:1000000;seq2Range=11396,11416;eValue=1.6929;cigar=16M1I4M;mutations=17A
seq1::900000:1000000	Stellar	eps-matches	24969	24988	95	-	.	seq2::900000:1000000;seq2Range=32201,32219;eValue=5.56469;cigar=18M1D1M;mutations=
seq1::300000:400000	Stellar	eps-matches	99065	99083	95	-	.	seq2::900000:1000000;seq2Range=32045,32064;eValue=5.56469;cigar=14M1I5M;mutations=15C
seq1::300000:400000	Stellar	eps-matches	13455	13473	95	-	.	seq2::900000:1000000;seq2Range=93116,93135;eValue=5.56469;cigar=6M1I13M;mutations=7G
seq1::300000:400000	Stellar	eps-matches	3067	3085	95	-	.	seq2::900000:1000000;seq2Range=35085,35104;eValue=5.56469;cigar=8M1I11M;mutations=9C
