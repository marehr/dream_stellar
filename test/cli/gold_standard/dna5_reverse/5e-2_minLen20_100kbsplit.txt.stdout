I/O options:
  database file   : 512_simSeq1_5e-2_100kbsplit.fa
  query file      : 512_simSeq2_5e-2_100kbsplit.fa
  alphabet        : dna5
  output file     : dna5_reverse/5e-2_minLen20_100kbsplit.txt
  output format   : txt

User specified parameters:
  minimal match length             : 20
  maximal error rate (epsilon)     : 0.05
  maximal x-drop                   : 10
  k-mer (q-gram) length            : 7
  search forward strand            : no
  search reverse complement        : yes

  verification strategy            : exact
  maximal number of matches        : 5000
  duplicate removal every          : 10000

Calculated parameters:
  s^min        : 10
  threshold    : 7
  distance cut : 20
  delta        : 16
  overlap      : 1

Loaded 10 query sequences.
Loaded 10 database sequences.

All matches matches resulting from your search have an E-value of: 
        5.56469 or smaller  (match score = 1, error penalty = -2)

Constructing index...

Aligning all query sequences to database sequence...
  seq1::0:100000 length=1000000 numMatches=500 errorRate=0.05 matchMinLength=50 matchMaxLength=200, complement
    # SWIFT hits      : 1390
    Longest hit       : 37
    Avg hit length    : 14
  seq1::100000:200000 length=1000000 numMatches=500 errorRate=0.05 matchMinLength=50 matchMaxLength=200, complement
    # SWIFT hits      : 1390
    Longest hit       : 32
    Avg hit length    : 14
  seq1::200000:300000 length=1000000 numMatches=500 errorRate=0.05 matchMinLength=50 matchMaxLength=200, complement
    # SWIFT hits      : 1448
    Longest hit       : 39
    Avg hit length    : 14
  seq1::300000:400000 length=1000000 numMatches=500 errorRate=0.05 matchMinLength=50 matchMaxLength=200, complement
    # SWIFT hits      : 1338
    Longest hit       : 36
    Avg hit length    : 14
  seq1::400000:500000 length=1000000 numMatches=500 errorRate=0.05 matchMinLength=50 matchMaxLength=200, complement
    # SWIFT hits      : 1410
    Longest hit       : 34
    Avg hit length    : 14
  seq1::500000:600000 length=1000000 numMatches=500 errorRate=0.05 matchMinLength=50 matchMaxLength=200, complement
    # SWIFT hits      : 1390
    Longest hit       : 35
    Avg hit length    : 14
  seq1::600000:700000 length=1000000 numMatches=500 errorRate=0.05 matchMinLength=50 matchMaxLength=200, complement
    # SWIFT hits      : 1414
    Longest hit       : 39
    Avg hit length    : 14
  seq1::700000:800000 length=1000000 numMatches=500 errorRate=0.05 matchMinLength=50 matchMaxLength=200, complement
    # SWIFT hits      : 1387
    Longest hit       : 31
    Avg hit length    : 14
  seq1::800000:900000 length=1000000 numMatches=500 errorRate=0.05 matchMinLength=50 matchMaxLength=200, complement
    # SWIFT hits      : 1449
    Longest hit       : 36
    Avg hit length    : 14
  seq1::900000:1000000 length=1000000 numMatches=500 errorRate=0.05 matchMinLength=50 matchMaxLength=200, complement
    # SWIFT hits      : 1376
    Longest hit       : 37
    Avg hit length    : 14

# Eps-matches     : 97
Longest eps-match : 23
Avg match length  : 20
